src/shared/services/IndexedDb.ts

```diff
@@ -251,6 +258,57 @@ export default class IndexedDb {
       );
     };
   }
+  private migrateModelNameSubscriptionsTableForV6(
```

Comment:
Might be a good idea to add test coverage

Reply:
I've added test coverage for this here: https://github.com/OneSignal/OneSignal-Website-SDK/pull/1179/commits/fcf13ee257092301c9f53a1ba7a9b6ae0d24775f, https://github.com/OneSignal/OneSignal-Website-SDK/pull/1179/commits/0633e7caabffc60e88c4e236f3ccd7dd614b238e

Code coverage report:
| File              | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                              |
|-------------------|---------|----------|---------|---------|------------------------------------------------|
| IndexedDb.ts      | 79.16   | 80       | 64.28   | 79.16   | 67-93,113,129,222,255,278,305,333-355,379,401-407,431-436 |

Reply:
ah, missed that. thanks